tests/scale/test_scale.py

```diff
@@ -0,0 +1,221 @@
+import concurrent.futures
+import json
+import os
+import time
+import unittest
+import requests
+
+from tests import logger
+
+
+def post_to_slack(message):
+    slack_url = 'https://hooks.slack.com/services/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'
```

Comment:
This should not be public. It should be in the secrets manager. Can you invalidate this one and create a new one and put it in the secrets manager?

Reply:
👍